hive_agent_client/client.py

```diff
@@ -89,7 +97,8 @@ async def get_entry_by_id(self, namespace: str, entry_id: str) -> Dict:
         try:
             return await get_entry_by_id(self.http_client, self.base_url, namespace, entry_id)
         except Exception as e:
-            raise Exception(f"Failed to get entry by ID: {e}")
+            logger.error(f"Failed to get entry {entry_id} from {namespace}: {e}")
+            raise Exception(f"Failed to get entry {entry_id} from {namespace}: {e}")
```

Comment:
@TianYao12 it looks like we're still doing it here in 2 other functions of the file